src/frontends/onnx/frontend/src/op/layer_norm.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/layer_norm.hpp"
+#include "ngraph/node.hpp"
+
+#include <algorithm>
```

Comment:
Looks like you're not using anything from the algorithms header as well as the ones below. Please clean up the includes section to the minimum required set of includes. This will improve the compilation time of this file.